src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -109,6 +110,40 @@ def config_filepath(self) -> Optional[str]:
         """Get the full Zowe z/OSMF Team Project Config filepath"""
         return self.project_config.filepath
 
+    @staticmethod
+    def get_env(
+        cfg: ConfigFile,
+    ) -> dict:
```

Comment:
The `get_env` method will map the environment variables to the profile properties

Reply:
The environment variable should be converted to whatever type is defined for the profile property in zowe.schema.json.

For example, `ZOWE_OPT_PORT` should be parsed as a number.
And `ZOWE_OPT_REJECT_UNAUTHORIZED` should be parsed as a boolean.